src/com/trainee/geetanjali/selenium/tests/InspectElementsAndApplyWaits.java

```diff
@@ -0,0 +1,75 @@
+package com.trainee.geetanjali.selenium.tests;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This class is used to perform the operations inorder to verify the logo and title and to add an item into cart
+ * and then checkout and to compare the both before and after checkout prices.
+ *
+ * @author geetanjali
+ */
+public class InspectElementsAndApplyWaits {
+    public static void main(String args[]) {
+        String title = "My Store";
+        //getting the directory of chromedriver by using getProperty method
+        String chromeDriverPath = System.getProperty("user.dir") + "\\drivers\\chromedriver-81.exe";
+        System.setProperty("webdriver.chrome.driver", chromeDriverPath);
+        WebDriver driver = new ChromeDriver();
+        //implicit waits are used as it takes time to load the page so by using implicit waits it will wait until the specified time through out the driver instance
+        driver.manage().timeouts().implicitlyWait(15, TimeUnit.SECONDS);
+        //here we are maximizing the window
+        driver.manage().window().maximize();
+        JavascriptExecutor jse = (JavascriptExecutor) driver;
+        driver.get("http://automationpractice.com/index.php");
+        //it finds the logo by using xpath as an locator
+        WebElement element = driver.findElement(By.xpath("//img[@class='logo img-responsive']"));
+        //isDisplayed method returns true if the element is displayed in the webpage.
+        if (element.isDisplayed()) {
+            System.out.println("logo is displayed on the home page");
+        } else {
+            System.out.println("logo is not displayed on the home page");
+        }
+        //getTitle method is used in order to get the title of a webpage
+        String actual = driver.getTitle();
+        System.out.println(driver.getTitle());
+        //it checks if the expected title and actual title ar equal or not
+        if (actual.equals(title)) {
+            System.out.println("Both expected title and actual title are same");
+        } else {
+            System.out.println("Both expected title and actual title are not same");
+        }
+        //finding a product by using xpath as an locator and to click on it
```

Comment:
Good that u mentioned comment. But it is good to mention the name that it belongs to EG: Womens xpath